### ISSUE TITLE
Feature Request: Alerta Support

### ISSUE THREAD

**Issue by hrzbrg** [Yelp/elastalert#553]
Hi,

it would be awesome to see Alerta as a build in alert output. It is an easy to use opensource alert dashboard with a simple API: http://alerta.readthedocs.io/en/latest/api/reference.html

Currently we made a workaround an let Elastalert trigger the Alerta CLI.

Regards
hrzbrg

**Comment by minhdanh** [comment]
:+1:

**Comment by jeff-cook** [comment]
Is anyone using this alert type? I see it has was added 2 years ago. Has anyone been successful with it?

Thanks

**Comment by jurgenhaas** [comment]
Yes, I'm using it successfully and the alerta specific components in each role look like this:

```
alert:
  - alerta
alerta_api_url: "https://alerta.example.com/api/alert"
alerta_api_key: "THEKEYFROMALERTA"
alerta_environment: "name"
alerta_use_qk_as_resource: true
alerta_text: "Text of alert"
```

This works really well, the only issue I'm having is #2310 where I would like to set severity dynamically so that depending on the ES records I could open or close an alerta item.

**Comment by moix** [comment]
I just integrated elastalert with alerta and first tests are working so far, so seems fine, thanks

**Comment by abhishekjiitr** [comment]
Yes, the Alerta integration works as mentioned in the documentation, have been using it for several months now.
